def
addNode
Expected to catch an exception but did not encounter any
three
two
getCause
 Make sure the message contains the nodes involved in the invalid transition to end
node [end]
asList
we
invokeForkJoin
getErrorCode
assertTrue
four
node [two]
dummyConf
end
testDecisionToEndForkJoinFailure

     *f->(2,3)
     *2->decision node->{4,end}
     *3->ok->j
     *3->fail->k
     *4->ok->j
     *4->fail->k
     *j->end
     
f
one
getMessage
j
assertEquals
k
kill
fail
contains
ex
parser
name
ErrorCode
Arrays
